packages/elements/src/number-field/__test__/number-field.step.test.js

```diff
@@ -220,4 +220,41 @@ describe('number-field/Step', function () {
       await expectValues(el, [-1, -2, -3, -4, -5], UP, -1.5);
     });
   });
+
+  describe('Long press spinner', function () {
+    const dispatchLongTapEvent = async (el, wait = 1000) => {
+      setTimeout(() =>
+        el.dispatchEvent(
+          new Event('tapstart', {
+            bubbles: true
+          })
+        )
+      );
+      await oneEvent(el, 'tapstart');
+      if (wait) {
+        await aTimeout(wait);
+      }
+      setTimeout(() =>
+        el.dispatchEvent(
+          new Event('tapend', {
+            bubbles: true
+          })
+        )
+      );
+      await oneEvent(el, 'tapend');
+    };
+
+    it('By tap, Step Up', async function () {
+      const el = await fixture('<ef-number-field></ef-number-field>');
+      await dispatchLongTapEvent(el.spinnerUpEl);
+      await elementUpdated(el);
```

Comment:
Does element really need wait to update?

Reply:
It doesn't need anymore. Removed.